lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/menu.rb'
+
+class Order
+
+attr_reader :list_of_dishes, :selected_items
+
+def initialize(m = Menu.new)
+  @list_of_dishes = m
+  @selected_items = []
+end
+
+def view_dishes
+  @item = @list_of_dishes.list
+end
+
+def select_dishes(item, quantity)
```

Comment:
I'm not clear on what the `i` variable in this method is doing for you. It seems like you could only ever enter one name of a dish at a time, so you shouldn't need a counter to keep the place of where you are in the array of selected dishes?